code/datums/spells/alien_spells/corrosive_acid_spit.dm

```diff
@@ -20,6 +20,11 @@
 	if(!plasma_check(200, user))
 		to_chat(user, "<span class='noticealien'>You don't have enough plasma to perform this action!</span>")
 		return
+	if(isliving(target) && target.acid_act(100, 50)) // If we don't do this, naked humans/simplemobs get absolutely nuked
```

Comment:
Can you please make this a variable and just change it if the target is living, to reduce duplicate code?

Reply:
I'm sorry but I've been looking at it and don't quite get what you mean, could you give some sort of example?